src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -40,72 +40,84 @@ export class JailAddComponent implements OnInit {
+                  devices for the jail. \
+                  A fully virtualized per-jail network stack will be \
+                  installed.'),
     },
     {
       type: 'checkbox',
       name: 'bpf',
       placeholder: T('Berkeley Packet Filter'),
-      tooltip: T('Check this for the jail to use the Berkeley Packet\
- Filter to data link layers in a protocol independent fashion. See <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=bpf&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">BPF(4)</a> for more details.'),
+      tooltip: T('Set to use the Berkeley Packet Filter (<a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=bpf&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">BPF(4)</a>) to data link layers in a \
+                  protocol independent fashion.'),
     },
     {
       type: 'input',
       name: 'ip4_addr',
       placeholder: T('IPv4 Address'),
-      tooltip: T('Configures network or internet access for the jail.\
- Type the IPv4 address for VNET and shared IP jails. Single interface\
- format: <b>interface|ip-address/netmask</b>. Multiple interface format:\
- <b>interface|ip-address/netmask,interface|ip-address/netmask</b>.\
- Example: <b>vnet0|192.168.0.10/24</b>'),
+      tooltip: T('Configure IPv4 networking or internet access for the \
+                  jail. Enter the IPv4 address for <a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=vnet&apropos=0&sektion=0&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">VNET(9)</a> and shared IP jails. \
+                  <br>Single interface format: <b>[interface|]\
+                  ip-address\
+                  [/netmask]</b>. <br>\
+                  Example: <b>vnet2|192.168.0.15/24</b> <br>\
+                  Multiple interface format: \
+                  <b>[interface|]ip-address[/netmask],[interface|]\
+                  ip-address[/netmask]</b>.<br>\
+                  Example: <b>192.168.0.10/24,vnet3|192.168.10.50</b>'),
       relation: [{
-        action: 'DISABLE',
-        when: [{
-          name: 'dhcp',
-          value: true,
-        }]
+      action: 'DISABLE',
+      when: [{
+        name: 'dhcp',
+        value: true,
+       }]
       }]
     },
     {
       type: 'input',
       name: 'defaultrouter',
-      placeholder: T('Default IPv4 Router'),
-      tooltip: T('Type <i>none</i> or a valid IP address. Setting this\
- property to anything other than <i>none</i> configures a default route\
- inside a <b>VNET</b> jail.'),
+      placeholder: T('Default IPv4 Route'),
+      tooltip: T('A valid IPv4 address to use as the default route. \
+                  <br>Enter <b>none</b> to configure the jail with \
+                  no IPv4 default route. <br>\
+                  <b>A jail without a default route will not be \
+                  able to access the network without additional \
+                  configuration.</b>'),
```

Comment:
This is not accurate: a jail without a default route will not be able to get outside the local network, but it will work on the local network.  Also, "without additional configuration" is really vague.  If there is something else the user needs, we should point them to it.  But really, this should say "Without a default route, a jail will not be able to access anything outside the local network."

Reply:
I said "without additional configuration" vague on purpose as it's an advanced topic outside the scope of a tooltip. An advanced user would know what the "additional configuration" needed is and why they would do it. This would be something to address in the user guide.
Not opposed to fixing, just explaining my reasoning.
I originally had "Access the outside world" but that didn't work for others.

Reply:
Clarified what happens with a jail when no default route is specified. No networking will work without additional advanced configuration, even local networking. There may be a reason to do such a thing but is an advanced use case like I stated above.

Reply:
Okay for the detail, I have not tested vnet. For that matter, do jails require vnet now?  Also, be careful to be consistent with terms.  I'm assuming you mean virtio, but might be mistaken.

As far as the "without additional configuration", we should not ever be vague.  That is frustrating for the user, telling them that we know how to solve their problem but are not going to tell them.

Overall, I suggest this section recommends always setting a default route for jails, then explains why (no network access with virtio, or limited to the local network otherwise), then link to the networking section of the handbook as a general solution.